health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/MyCampaignNew.js

```
@@ -13,40 +13,64 @@
   const [tabData, setTabData] = useState(
     myCampaignConfigNew?.myCampaignConfigNew?.map((configItem, index) => ({
       key: index,
       label: configItem.label,
-      active: index === 0,
+      active: index === 0 ? true : false,
     }))
   );
+  const [isConfigReady, setIsConfigReady] = useState(false);
+
+   useEffect(() => {
+    const savedIndex = parseInt(sessionStorage.getItem("HCM_SELECTED_TAB_INDEX")) || 0;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)